extensions/prisma-cli-commands/src/data/index.ts

```diff
@@ -0,0 +1,418 @@
+import { Icon } from "@raycast/api";
+import { Category, Section, PrismaCommand } from "../types";
+import { details } from "./details";
+
+export const prismaSections: Section[] = [
+  {
+    title: "Common",
+    category: Category.COMMON,
+    tintColor: "#4757e6",
+    icon: Icon.Terminal,
+  },
+  {
+    title: "DB",
+    category: Category.DB,
+    tintColor: "#24db67",
+    icon: Icon.HardDrive,
+  },
+  {
+    title: "Migrate",
+    category: Category.MIGRATE,
+    tintColor: "#f3538b",
+    icon: Icon.Cloud,
+  },
+  {
+    title: "Studio",
+    category: Category.STUDIO,
+    tintColor: "#0d9aff",
+    icon: Icon.AppWindowSidebarLeft,
+  },
+];
```

Comment:
```suggestion
export const prismaSections: Section[] = [
  {
    title: "Common",
    category: Category.COMMON,
    tintColor: Color.Purple,
    icon: Icon.Terminal,
  },
  {
    title: "DB",
    category: Category.DB,
    tintColor: Color.Green,
    icon: Icon.HardDrive,
  },
  {
    title: "Migrate",
    category: Category.MIGRATE,
    tintColor: Color.Magenta,
    icon: Icon.Cloud,
  },
  {
    title: "Studio",
    category: Category.STUDIO,
    tintColor: Color.Blue,
    icon: Icon.AppWindowSidebarLeft,
  },
];
```
Would this be okay? Since Raycast can be themed it's much better to use variables for colours

Reply:
@pernielsentikaer It makes sense. Updated the colors to Raycast theme colors as you suggested.